projects/storefrontapp-e2e-cypress/cypress/integration/b2b/my-account/b2b-replenishment-order-details-flow.e2e-spec.ts

```diff
@@ -16,9 +16,11 @@ describe('Replenishment order details', () => {
 
   it('should be able to login as a b2b user', () => {
     loginB2bUser();
+    cy.saveLocalStorage();
   });
 
   it('should be able to cancel a replenishment order details', () => {
+    cy.restoreLocalStorage();
```

Comment:
Can we merge these 2 tests into 1. I think there is no need just test "should be able to login as a b2b user".